app/src/main/kotlin/org/eurofurence/connavigator/ui/FragmentViewDealer.kt

```diff
@@ -128,38 +134,50 @@ class DealerUi : AnkoComponent<ViewGroup> {
                             padding = dip(10)
 
                             setTextAppearance(ctx, android.R.style.TextAppearance_Medium_Inverse)
-
                         }
                     }
 
                     textView {
                         text = "About the Artist"
-                        padding = dip(15)
+                        setPadding(dip(25), dip(20), dip(25), dip(0))
                         setTextAppearance(ctx, R.style.TextAppearance_AppCompat_Large)
-
                     }
 
-                    aboutArtist = markdownView { }
-
-                    artPreview = photoView {
-                        lparams(matchParent, dip(400))
-                        scaleType = ImageView.ScaleType.FIT_CENTER
-                        backgroundResource = R.color.primary
+                    aboutArtist = markdownView {
+                        lparams {
+                            margin = dip(15)
```

Comment:
Can you give me an example on how you would do that here? I'll fix the rest accordingly.